pkg/plan/resource/kubeadm_init.go

```diff
@@ -126,7 +126,7 @@ func (ki *KubeadmInit) Apply(ctx context.Context, runner plan.Runner, diff plan.
 	}
 
 	var dns *kubeadmapi.DNS = nil
-	dnsAD, found := ki.AssetDescriptions["DNS"]
+	dnsAD, found := ki.AssetDescriptions["Coredns"]
```

Comment:
As mentioned above, we said we were going to use "DNS" as that's what the k8s go code uses...